modules/default-requirements-central-services.adoc

```diff
@@ -28,12 +28,11 @@ Central DB requires persistent storage in the cluster where Central is installed
 [NOTE]
 ====
 You can use a hostPath volume for storage only if all your hosts (or a group of hosts) mount a shared file system, such as an NFS share or a storage appliance.
-Otherwise, your data is only saved on a single node. Red{nbsp}Hat does not
-recommend using a hostPath volume.
+Otherwise, your data is only saved on a single node. Red{nbsp}Hat does not recommend using a hostPath volume.
 ====
 * Use Solid-State Drives (SSD) for best performance.
 However, you can use another storage type if you do not have SSDs available.
-* If you use a web proxy or firewall, you must configure bypass rules to allow traffic for the `definitions.stackrox.io` domain and enable {product-title} to trust your web proxy or firewall. Otherwise, updates for vulnerabilities will fail.
+* If you use a web proxy or firewall, you must configure bypass rules to allow traffic for the `definitions.stackrox.io` domain and enable {product-title-short} to trust your web proxy or firewall. Otherwise, updates for vulnerability definitions will fail. You must also ensure that incoming connections from Sensor on secured clusters to Central on port 443 are possible.
```

Comment:
`vulnerability definitions` -> I actually preferred just `vulnerabilities`. I don't know who decided on "definitions" years ago, but I hate it 😂